kratos/processes/time_averaging_process.cpp

```diff
@@ -396,14 +392,13 @@ void TimeAveragingProcess::CalculateTimeIntegratedNonHistoricalQuantity(
         this->GetTimeAveragingMethod<TDataType>();
 
     const int number_of_nodes = rContainer.size();
-#pragma omp parallel for
-    for (int i_node = 0; i_node < number_of_nodes; ++i_node)
-    {
-        auto& r_container_element = *(rContainer.begin() + i_node);
+
+    IndexPartition<std::size_t>(number_of_nodes).for_each([&](std::size_t Index){
```

Comment:
here you can also use the block_for_each I think to directly get the container_element

Reply:
I tried using that, block_for_each(As Shown).. But we dont have EntityContainerType right?? 
![image](https://user-images.githubusercontent.com/46954821/108523495-43b99f80-72ce-11eb-9145-f8eb6ea52c7c.png)
Using above gave me an error **"No member named GetValue()....."**
Am I making any mistake ??

Reply:
hm not sure
anyway should be ok